content/english/arduino/1-introduction-building-with-ardunio/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Introduction - Building with Arduino"
+description: ""
+draft: true

```

Comment:
Can you add prereq and difficulty here as well? just like other workshops?

Reply:
Fixed